### ISSUE TITLE
Saving edited PDFs on Android not seamless

### ISSUE THREAD

**Issue by link6155** [FBarrca/obsidian-handwritten-notes#5]
Since the PDFs are opened in editors through the share menu, the file's original location isn't exposed and so the editor can't save changes back into the Obsidian Vault without extra steps. WPS works OK because it'll ask you to choose your save location, but some editors don't do this (e.g. Neoreader for Boox tablets). Is there a different way to get PDFs opened up in external editors?

**Comment by FBarrca** [comment]
Yes, this is an issue that I am aware. Because of how most editors in Android are coded, the share menu creates a copy of the file. In the README, you have a list of editors that don't do that and do work. 
For me, Xodo is the best by far, you can install it in the App Store. You can use it without a subscription but some minor features are locked. If you are comfortable sideloading an APK, you can search online for an outdated version that does 99% of the things it does now and has no paywall.
